backend/src/backend/filterlog.clj

```diff
@@ -20,8 +20,12 @@
 (defn get-filtered-syslog
   "Retrieve and filters data from the database table 'system_log'
   based on input."
-  [{:keys [device_id adaption_id description date date_from date_to]}]
-  (def querystr "SELECT * FROM system_log WHERE")
+  [{:keys [device_id adaption_id adaption_type description date date_from date_to]}]
+  (def querystr "SELECT system_log.system_log_id, system_log.device_id, system_log.adaption_id, adaption.adaption_type, adaption.adaption_description, adaption.config_id, system_log.description, system_log.created
```

Comment:
Enig i hvordan du gjør join her for for produksjonsformål. Fwiw, vil du kunne få samme resultat med ' select foo, bar from system_log natural join adaption '

Reply:
Men igjen, natural join er ganske antipattern i produksjon